src/file.js

```diff
@@ -35,11 +35,15 @@ function normalizePath(str) {
 
 /**
  * Check whether a resource is external or not
- * @param {string} href Path
+ * @param {string|Vinyl} href Path
  * @returns {boolean} True if the path is remote
  */
 function isRemote(href) {
-  return /(^\/\/)|(:\/\/)/.test(href) && !href.startsWith('file:');
+  if (isVinyl(href)) {
+    return false;
```

Comment:
I ended up adding a separate check for `Vinyl`. I'm not sure if the test is right; if it is then this check makes sense. If not, we should change the test @bezoerb

Reply:
The `isRemote` function shouldn't intentionally receive a vinyl object. Until we find out where the vinyl is coming from the test should be something like this: 
```js
if (isVinyl(href)) {
   // Check custom attribute set in the vinylize function
  return Boolean(href.remote)
}
```